packages/react-docs/RELEASE-NOTES.md

```diff
@@ -3,6 +3,77 @@ id: Release notes
+
+### Components
+- **Badge:**
+  - Added `pf-screen-reader` for screen reader text ([#8361](https://github.com/patternfly/patternfly-react/pull/8361))
+- **Calendar month:**
+  - Provided explicit guidance for labeling an inline calendar month ([#8375](https://github.com/patternfly/patternfly-react/pull/8375))
+- **Clipboard copy:**
+  - Added `removeFindDomNode` ([#8371](https://github.com/patternfly/patternfly-react/pull/8371))
+- **Dropdown:**
+  - Added `itemId` to `DropdownItem` props to the next version of the dropdown ([#8356](https://github.com/patternfly/patternfly-react/pull/8356))
+- **Menu:**
+  - Added drilldown filter demo, also added flag to support demo and fix some keyboard interaction ([#8405](https://github.com/patternfly/patternfly-react/pull/8405))
+  - Removed `li` wrapper from breadcrumb example ([#8433](https://github.com/patternfly/patternfly-react/pull/8433))
+  - Updated breadcrumb drilldown example ([#8385](https://github.com/patternfly/patternfly-react/pull/8385))
+- **Navigation:**
+  - Fixed navigation flyout accessibility ([#8279](https://github.com/patternfly/patternfly-react/pull/8279))
+- **Pagination:**
+  - Added support for insets ([#8412](https://github.com/patternfly/patternfly-react/pull/8412))
+  - Fixed filter not applying in sticky example ([#8133](https://github.com/patternfly/patternfly-react/pull/8133))
+- **Panel:**
+  - Updated component unit tests to meet new RTL standards ([#8346](https://github.com/patternfly/patternfly-react/pull/8346))
+- **Search input:**
+  - Updated advanced search menu to allow spaces in search form field values (#8369) ([#8372](https://github.com/patternfly/patternfly-react/pull/8372))
+- **Sidebar:**
+  - Updated tests to new react testing library standards ([#8342](https://github.com/patternfly/patternfly-react/pull/8342))
+- **Tabs:**
+ - Converted examples to TypeScript ([#8337](https://github.com/patternfly/patternfly-react/pull/8337))
+- **Time picker:**
+  - Removed redundant `onBlur` callback ([#8366](https://github.com/patternfly/patternfly-react/pull/8366))
+- **Tooltip:**
+  - Converted examples to TypeScript ([#8052](https://github.com/patternfly/patternfly-react/pull/8052))
+- **Truncate:**
+  - Added `removeFindDomNode` ([#8371](https://github.com/patternfly/patternfly-react/pull/8371))
+- **Navigation:**
+  - Fixed navigation flyout accessibility ([#8279](https://github.com/patternfly/patternfly-react/pull/8279))
+- **Wizard:**
+  - Allowed for async function types for navigation & `isCollapsible` wizard step prop for the next version of the wizard ([#8332](https://github.com/patternfly/patternfly-react/pull/8332))
+  - Converted examples to TypeScript ([#8287](https://github.com/patternfly/patternfly-react/pull/8287))
+
+### Access console extension
+- Prevented running initialization code more than once (#8373) ([#8374](https://github.com/patternfly/patternfly-react/pull/8374))
```

Comment:
A link is missing on this PR number

Reply:
The link works for me.  I removed the issue number that was in commit message.

Reply:
+1 - same note from Search input above about the issue number in the issue title displaying right next to the issue number linking to the PR.  Your note addresses this, but it looks like the text is still there fyi
```suggestion
- Prevented running initialization code more than once ([#8374](https://github.com/patternfly/patternfly-react/pull/8374))
```